src/main/java/gregtech/loaders/recipe/CraftingRecipeLoader.java

```diff
@@ -90,7 +90,7 @@ private static void loadCraftingRecipes() {
         for(Material material : new Material[] {Materials.Lapis, Materials.Lazurite, Materials.Sodalite}) {
             String recipeName = "lapotron_crystal_" + material.toString();
             ModHandler.addShapedEnergyTransferRecipe(recipeName, MetaItems.LAPOTRON_CRYSTAL.getStackForm(),
-                Ingredient.fromStacks(MetaItems.ENERGY_CRYSTAL.getStackForm()), false,
+                Ingredient.fromStacks(MetaItems.LAPOTRON_CRYSTAL.getStackForm()), MetaItems.LAPOTRON_CRYSTAL.getChargeAmount(), false,
```

Comment:
You need to pass in info from which items it is crafted so it can transfer energy from them - right now it just wastes energy from ENERGY_CRYSTAL

Reply:
This has been fixed.